gitbootcamp.rst

```diff
@@ -203,6 +203,33 @@ from the `core-workflow  <https://github.com/python/core-workflow>`_
 repository to backport the commit.
 
 
+.. _git_from_mercurial:
+
+Applying a Patch from Mercurial to Git
+--------------------------------------
+
+Scenario:
+
+- A Mercurial patch exists but there is no pull request for it.
+
+Solution:
+
+1. Download the patch locally.
+
+2. Create a new branch and switch to it.
+
+3. For Mac and Unix::
+
+   $ patch -p1 < /path/to/issueNNNN.patch
```

Comment:
I'm trying out both `git apply` and `patch` to apply the patch from this issue: http://bugs.python.org/issue12815

I downloaded PatchProxy.diff, and then ran `patch -p1 < PatchProxy.diff`. There are some conflicts, but at least I could apply the patch as is.

I was not able to do `git apply PatchProxy.diff` because the patch was not created using `--git` option.

Maybe this is why the `patch` command was recommended in the first place? 🤔 
If it's availble in the OS, it's more straightforward to use.

With `git apply` we first need to make sure that the patch was made with `--git` option.
Or am I missing something?